### PR TITLE
added tests for type annotation checking and removing the unnecessary…

### PR DESCRIPTION
**Pull request by Nattalli**
… comment

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -218,3 +222,81 @@ def test_methods_declared_in_inherited_classes(class_, methods, length):
     assert (
             [parsed_class.body[0].body[it].name for it in range(length)] == methods
     ), f"Only {methods} should be defined inside class '{class_.__name__}'"
+
+
+def test_removed_comment():
+    import app
```
Comment (DmytroSvirsa) [inline tests/test_main.py:228]: not sure that that way to import is the best practice

File: tests/test_main.py [hunk 2]
```diff
@@ -218,3 +222,81 @@ def test_methods_declared_in_inherited_classes(class_, methods, length):
...
+    with open(app.main.__file__, "r") as f:
```
Comment (Abnormaltype) [inline tests/test_main.py:229]: f - a bad name

File: tests/test_main.py [hunk 3]
```diff
@@ -218,3 +222,81 @@ def test_methods_declared_in_inherited_classes(class_, methods, length):
...
+        assert "# write your code here" not in file_content, ("You have to"
+               " remove the unnecessary comment '# write your code here'")
+
+
+@pytest.mark.parametrize(
+    "function,result",
+    [
+        (
+                BaseRobot.__init__,
+                {'coords': typing.Optional[list[int]],
```
Comment (Abnormaltype) [inline tests/test_main.py:240]: why ''?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by Abnormaltype — APPROVED** [review]
LGTM

**Pull request merged**